### PR TITLE
Initial version of install instructions

### LINKED ISSUE #13: Installation instructions with Trustchain
Add install instructions (including interaction with Trustchain server).

### PR DESCRIPTION
**Pull request by lukehare**
Closes #13 
Add trustchain-mobile app installation & development instructions Mac.

### CODE REVIEW

File: install_trustchain_mobile.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+[`cargo ndk`](https://crates.io/crates/cargo-ndk/) provides the simplest method to build for Android targets. This is installed with:
+```
+cargo install cargo-ndk
+```
+
+### 8. Build DIDKit targets
+This step provides the builds that provide the DIDKit FFI functionality.
+
+A Rust installation is needed for this step. If necessary, follow the [official instructions](https://www.rust-lang.org/tools/install) including adding `~/.cargo/bin` to your path. 
+
```
Comment (sgreenbury) [inline install_trustchain_mobile.md:119]:
```suggestion
A Rust installation (at least 1.68.0) is needed for this step. If necessary, follow the [official instructions](https://www.rust-lang.org/tools/install) including adding `~/.cargo/bin` to your path. 

```

**Comment by thobson88** [comment]
In step 2, should there be a `git clone` command for didkit? Seems to be missing and I'm wondering if there's a particular branch I should be on.

**Comment by lukehare** [comment]
> In step 2, should there be a `git clone` command for didkit? Seems to be missing and I'm wondering if there's a particular branch I should be on.

Yep, this was a typo - we were accidentally cloning the ssi repo twice instead of didkit + ssi. I just pushed a correction.

**Comment by thobson88** [comment]
I'm getting this error in Step 8 (running the command from `/Users/thobson/spruceid/didkit`):
```bash
thobson@521-D7M932T9F2 didkit % make -C lib ../target/test/flutter.stamp
cd flutter && LD_LIBRARY_PATH=/Users/thobson/spruceid/didkit/lib/flutter \
                flutter --suppress-analytics test
Running "flutter pub get" in flutter...
Resolving dependencies...
  async 2.10.0 (2.11.0 available)
  characters 1.2.1 (1.3.0 available)
  collection 1.17.0 (1.18.0 available)
  ffi 2.0.2 (2.1.0 available)
  js 0.6.5 (0.6.7 available)
  matcher 0.12.14 (0.12.16 available)
  material_color_utilities 0.2.0 (0.8.0 available)
  meta 1.8.0 (1.9.1 available)
  pedantic 1.11.1 (discontinued replaced by lints)
  source_span 1.9.1 (1.10.0 available)
  stack_trace 1.11.0 (1.11.1 available)
  stream_channel 2.1.1 (2.1.2 available)
  test_api 0.4.17 (0.6.1 available)
Got dependencies!
Resolving dependencies in ./example...
Resolving dependencies in ./example failed. For details run `flutter pub get --directory ./example`
pub get failed
command: "/Users/thobson/flutter/bin/cache/dart-sdk/bin/dart __deprecated_pub --color --directory . get
--example"
pub env: {
  "FLUTTER_ROOT": "/Users/thobson/flutter",
  "PUB_ENVIRONMENT": "flutter_cli:verify:test",
  "PUB_CACHE": "/Users/thobson/.pub-cache",
}
exit code: 1
make: *** [../target/test/flutter.stamp] Error 1
```

**Pull request merged**